app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -260,33 +358,136 @@ class Exposure extends React.Component {
-              maxDate={moment()
-                .add(30, 'days')
-                .format('YYYY-MM-DD')}
-              onChange={date => this.handleDateChange('symptom_onset', date)}
-              placement="bottom"
-              isInvalid={!!this.state.errors['symptom_onset']}
-              customClass="form-control-lg"
-              ariaLabel="Symptom Onset Date Input"
-            />
-            <Form.Control.Feedback className="d-block" type="invalid">
-              {this.state.errors['symptom_onset']}
-            </Form.Control.Feedback>
+          <Form.Group as={Col} sm="24" md="12" xl="9" controlId="symptom_onset" className="mb-2">
+            <Row>
+              <Col className="mb-2">
+                <Form.Label className="nav-input-label">SYMPTOM ONSET DATE{schema?.fields?.symptom_onset?._exclusive?.required && ' *'}</Form.Label>
+                <DateInput
+                  id="symptom_onset"
+                  aria-label="Symptom Onset Date"
+                  date={this.state.current.patient.symptom_onset}
+                  minDate={'2020-01-01'}
+                  maxDate={moment()
+                    .add(30, 'days')
+                    .format('YYYY-MM-DD')}
+                  onChange={date => this.handleDateChange('symptom_onset', date)}
+                  placement="bottom"
+                  isInvalid={!!this.state.errors['symptom_onset']}
+                  customClass="form-control-lg"
+                  ariaLabel="Symptom Onset Date Input"
+                  isClearable={!this.props.symptomatic_assessments_exist}
+                />
+                <Form.Control.Feedback className="d-block" type="invalid">
+                  {this.state.errors['symptom_onset']}
+                </Form.Control.Feedback>
+              </Col>
+            </Row>
+            <Row>
+              <Col md="auto" className="mb-2">
+                <Form.Check
+                  size="lg"
+                  label={`NO SYMPTOM HISTORY${schema?.fields?.no_symptom_history?._whitelist?.list?.has(true) ? ' *' : ''}`}
+                  id="no_symptom_history"
+                  className="ml-1 d-inline"
+                  checked={!!this.state.current.patient.no_symptom_history}
+                  onChange={this.handleChange}
+                  disabled={this.props.symptomatic_assessments_exist}
+                />
+                <InfoTooltip
+                  tooltipTextKey={this.props.symptomatic_assessments_exist ? 'noSymptomHistoryDisabled' : 'noSymptomHistory'}
+                  location="right"></InfoTooltip>
+              </Col>
+            </Row>
+            <Row>
+              {this.state.current.first_positive_lab && (
+                <Col>
+                  <div className={this.state.current.patient.no_symptom_history ? '' : 'disabled-first-positive-lab'}>
+                    <div className="section-header">
+                      <Form.Label className="nav-input-label">FIRST POSITIVE LAB RESULT</Form.Label>
```

Comment:
@tstrass I agree, @sgober ?

Reply:
It would also help significantly if the FIRST POSITIVE LAB RESULT section were indented (possibly to align with the NO SYMPTOM HISTORY label) and the NOTES label alignment were fixed (it's too far to the right). Either way I think would be fine.

Reply:
@tstrass, yes it was intentionally moved because of resizing on smaller screens.  Before it was below the case status dropdown which would cause things to be out of order if they resize.  Keeping it here is definitely the way to go in regards to that

The alignment is def a tad off - that should be fixed.  You can also add a bit of padding below the "First Positive Lab Result" section so there is a clear difference in sections

Reply:
this has been fixed here: https://github.com/SaraAlert/SaraAlert/pull/747/commits/54e53104c56db77f796fe3eba5f0cf8fc89632d8

Reply:
Seems if I go narrow enough now, the lab result pops up above the symptom onset date (Firefox). If it's easier to make it so it's always just below the no symptom history checkbox, that's fine with me as long as the alignment is fixed.

![image](https://user-images.githubusercontent.com/8235974/114071355-05247800-986f-11eb-9988-04cbc6a05bbf.png)

Reply:
ah good catch, it's because I forgot to change `sm` to `xs` on line 431, I just updated it with the fix